### PR TITLE
Added a pattern for homepage

### PR DESCRIPTION
**Pull request by Imran92**
Implements https://github.com/Automattic/sensei/issues/6074

## Changes proposed

- Added a pattern with relevant font, template and styles for theme homepage.

## Testing instructions

- Create a new page
- In the editor, change the template of the page to **Page - No Title**
- Now clear if there's any block on the page and click on the inserter
- Click on patterns Tab
- Click on the dropdown for pattern categories and select "Sensei LMS" from there
- Make sure the **Landing Page** pattern is there, select it.
- Click on the pattern, save the page, and go to the frontend. Make sure the design matches

## Screenshots and Videos

**Desktop**

https://user-images.githubusercontent.com/6820724/196554308-2cae64a0-d273-43dc-8dcb-2c471386a8cc.mov

**Mobile**

https://user-images.githubusercontent.com/6820724/196554366-4cdb09f9-40d9-4260-91dd-3b971af23be0.mov

### CODE REVIEW

File: patterns/homepage.php [hunk 1]
```diff
@@ -0,0 +1,230 @@
+<?php
+/**
+ * Title: Homepage
+ * Slug: course/homepage
+ * Inserter: yes
+ * Categories: sensei-lms
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"className":"course-homepage-pattern-parent-group","layout":{"inherit":false,"wideSize":"","contentSize":""}} -->
```
Comment (gabrielcaires) [inline patterns/homepage.php]: I think we should split this pattern into small parts to be reused on the other screens.
My suggestion is
- Grow your writing skills with the course is a pattern
- Course list 
- Blog Post
-The ` Keep track of the latest news and lessons. Every week in your inbox.`
-Testimonial

And we should use the home pattern only to connect the small patterns.
We have a lot of screens using the same blocks so we should not rewrite the same instructions

WDT? 

Cc @donnapep

File: patterns/homepage.php [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Spotify</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:heading {"textAlign":"center","className":"has-sorts-mill-goudy-font-family","fontSize":"5x-large"} -->
+<h2 class="has-text-align-center has-sorts-mill-goudy-font-family has-5-x-large-font-size">COURSE</h2>
```
Comment (gabrielcaires) [inline patterns/homepage.php]: I think we should move the footer to the template part instead of keeping it here.

File: style.css [hunk 3]
```diff
@@ -227,3 +227,80 @@ body .is-layout-flow .wp-block-group {
...
+	font-family: var(--wp--preset--font-family--system);
+	line-height: 145%;
+	padding: 3px 6px;
+}
+
+.wp-block-post-author__avatar img {
+	border: 1px solid var(--wp--preset--color--tertiary);
+	border-radius: 4px;
+}
+}
```
Comment (gabrielcaires) [inline style.css:368]: is this  extra `}` correct?

File: theme.json [hunk 4]
```diff
@@ -228,6 +262,18 @@
 					}
 				}
 			},
+			"core/group": {
+				"elements": {
+					"h2": {
+						"typography": {
+							"fontFamily": "var(--wp--preset--font-family--league-gothic)"
```
Comment (gabrielcaires) [inline theme.json]: The h2 font family is already declared here https://github.com/Automattic/sensei-theme/blob/trunk/theme.json#L562.
So I think this line is unnecessary.

File: patterns/homepage.php [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"primary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-primary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-gothic-font-family"} -->
+<h3 class="has-gothic-font-family" style="text-transform:uppercase">pages</h3>
+<!-- /wp:heading -->
+
+<!-- wp:group {"className":"has-eb-garamond-font-family","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group has-eb-garamond-font-family"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size">About</p>
```
Comment (gabrielcaires) [inline patterns/homepage.php]: I think these links are actually page links, should we use a nav block?

File: functions.php [hunk 6]
```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
...
+	register_block_style(
+		'core/image',
+		array(
+			'name'  => 'course-image-border',
+			'label' => __( 'Image Border', 'course' ),
+		)
+	);
+
+	$block_pattern_categories = array(
+		'sensei-lms' => array( 'label' => __( 'Sensei LMS', 'course' ) )
```
Comment (gabrielcaires) [inline functions.php]: Why do we need to translate `Sensei LMS` ?

File: functions.php [hunk 7]
```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
...
+	 *     @type array[] $properties {
+	 *         An array of block category properties.
+	 *
+	 *         @type string $label A human-readable label for the pattern category.
+	 *     }
+	 * }
+	 */
+	$block_pattern_categories = apply_filters( 'course_block_pattern_categories', $block_pattern_categories );
+
+	foreach ( $block_pattern_categories as $name => $properties ) {
```
Comment (gabrielcaires) [inline functions.php]: Could you please provide more context about why we need to use PHP only to register the pattern?
it theoretically should be automatically, rigth?

File: patterns/homepage.php [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+<div class="wp-block-group alignfull course-home-page-responsive-padding has-foreground-color has-primary-background-color has-text-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:spacer {"height":"10px"} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"backgroundColor":"primary","textColor":"tertiary","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"center"}} -->
+<div class="wp-block-group has-tertiary-color has-primary-background-color has-text-color has-background"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"tertiary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-tertiary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-league-gothic-font-family"} -->
+<h3 class="has-league-gothic-font-family" style="text-transform:uppercase">COURSE LIST</h3>
```
Comment (gabrielcaires) [inline patterns/homepage.php]: This string should be translated

File: patterns/homepage.php [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph {"className":"has-system-font-family course-font-color-tertiary","fontSize":"xx-small"} -->
+<p class="has-system-font-family course-font-color-tertiary has-xx-small-font-size"><a href="http://test.test">Explore all courses</a></p>
```
Comment (gabrielcaires) [inline patterns/homepage.php]: Translated string

**Review by gabrielcaires — CHANGES_REQUESTED** [review]
I left some comments about splitting the pattern into small ones.

File: patterns/homepage.php (hunk 1 above)
Comment (donnapep) [reply patterns/homepage.php]: @Automattic/theam Any thoughts on this, or on this PR in general?

File: style.css (hunk 3 above)
Comment (scruffian) [reply style.css:368]: Do you need this CSS or can you put the settings in the blocks? If the settings are inline in the block then its easier for users to change them...

File: functions.php [hunk 10]
```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
 			'label' => __( 'Button', 'course' ),
 		)
 	);
+	register_block_style(
```
Comment (scruffian) [inline functions.php]: Why do we need block styles for this?

File: theme.json [hunk 11]
```diff
@@ -156,11 +180,21 @@
 					"size": "clamp(3rem, 5vw + 1.5rem, 6.029rem)",
 					"slug": "3x-large"
 				},
+				{
+					"name": "3.5X Large",
+					"size": "clamp(64px, calc(4rem + ((1vw - 4.8px) * 6.3889)), 110px);",
```
Comment (MaggieCabrera) [inline theme.json:204]: you might want to consider using the [built-in tools](https://make.wordpress.org/core/2022/05/27/block-font-sizes-and-fluid-typography/) for fluid typography

File: patterns/homepage.php (hunk 1 above)
Comment (Imran92) [reply patterns/homepage.php]: Thanks for the suggestion @gabrielcaires , I've broken it into reusable smaller patterns here https://github.com/Automattic/sensei-theme/pull/44/commits/0bf443111747fd27d9bc2a24346d753de739cc03, can you take a look?

File: patterns/homepage.php (hunk 2 above)
Comment (Imran92) [reply patterns/homepage.php]: Right, moved it here https://github.com/Automattic/sensei-theme/pull/44/commits/0f0b9e3c852b17e88f97b7e7269bea46067189cc

File: style.css (hunk 3 above)
Comment (Imran92) [reply style.css:368]: @gabrielcaires thanks for noticing, I've removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/b87083c1c01f4c84fdced59affdfc65b028f13f9 :)

Comment (Imran92) [reply style.css:368]: @scruffian thanks! :) I've removed all the styles which we could apply inline from here in this update https://github.com/Automattic/sensei-theme/pull/44/commits/0ac98bb70032bc77b6b62697655cb1e851b378e1

File: theme.json (hunk 4 above)
Comment (Imran92) [reply theme.json]: Thanks @gabrielcaires , removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/139ed3b508555de4fa89647526be8dc6edadfc81

File: patterns/homepage.php (hunk 5 above)
Comment (Imran92) [reply patterns/homepage.php]: Thanks @gabrielcaires , I looked a bit into using nav block here. It seems nav block is more suitable for the navbar and it's mostly used in that context. I think maybe we can keep it this way for now

File: functions.php (hunk 6 above)
Comment (Imran92) [reply functions.php]: Removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/73331765814cfc083618f8a463f16acd60d6cde1

File: functions.php (hunk 7 above)
Comment (Imran92) [reply functions.php]: Thanks @gabrielcaires , it is update here https://github.com/Automattic/sensei-theme/pull/44/commits/73331765814cfc083618f8a463f16acd60d6cde1

File: patterns/homepage.php (hunk 8 above)
Comment (Imran92) [reply patterns/homepage.php]: Translated here https://github.com/Automattic/sensei-theme/pull/44/commits/efb98493e7afa7a1576dbb5b291a0b5cd2a12328

File: patterns/homepage.php (hunk 9 above)
Comment (Imran92) [reply patterns/homepage.php]: Translated here https://github.com/Automattic/sensei-theme/pull/44/commits/efb98493e7afa7a1576dbb5b291a0b5cd2a12328

File: functions.php (hunk 10 above)
Comment (Imran92) [reply functions.php]: Thanks for taking a look @scruffian, I've removed the block style [here](https://github.com/Automattic/sensei-theme/pull/44/commits/a3cde4a814d1a42e2c865715a54c7a9d9a3832f5)

File: theme.json (hunk 11 above)
Comment (Imran92) [reply theme.json:204]: I've just checked it and it looks awesome! Thanks for the great suggestion @MaggieCabrera ! As we have multiple font sizes using clamp, I'll open a separate PR changing the clamps to fluids and check to make sure nothing breaks

**Review by gabrielcaires — CHANGES_REQUESTED** [review]
@Imran92 I think the footer is a template part, so it will update all other templates with the correct footer

**Comment by donnapep** [comment]
Looks like the block pattern and template use "Homepage" instead of "Home Page" as per the issue.

**Comment by donnapep** [comment]
Just to be sure I understand correctly (I think the PR description may be a bit stale), I need to add each pattern individually to the page? Is it possible to add the patterns automatically when I select the Home Page template?

Also, I don't think we need to preface each pattern with "Home Page", as it implies those patterns can only be used on a home page.

File: patterns/blog-posts.php [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+<!-- wp:group {"textColor":"primary","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"top"}} -->
+<div class="wp-block-group has-primary-color has-text-color"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}},"border":{"top":{"width":"0px","style":"none"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"width":"1px","color":"var:preset|color|primary"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--primary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}}} -->
+<h3 style="text-transform:uppercase">Blog posts</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph {"fontSize":"xx-small","fontFamily":"system"} -->
+<p class="has-system-font-family has-xx-small-font-size"><a href="http://test.test">Visit the blog</a></p>
```
Comment (donnapep) [inline patterns/blog-posts.php]: Should we just use `#` for this URL?

File: patterns/course-list-homepage.php [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"fontFamily":"league-gothic"} -->
+<h3 class="has-league-gothic-font-family" style="text-transform:uppercase">COURSE LIST</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|tertiary"}}}},"fontSize":"xx-small","fontFamily":"system"} -->
+<p class="has-link-color has-system-font-family has-xx-small-font-size"><a href="http://test.test">Explore all courses</a></p>
```
Comment (donnapep) [inline patterns/course-list-homepage.php]: I'm guessing this should probably link to the page set in Sensei > Settings > Course Archive Page.

**Comment by donnapep** [comment]
Looks like the block pattern and template are named "Homepage" instead of "Home Page".  I don't think we need to prefix each pattern with "Home Page", as it implies those patterns can only be used on a home page.

The testimonial image is not working for me, as the URL appears to be hardcoded.

I find the pattern behaviour a little strange. Here's what I did:

- Created a page I called "Home".
- Set the template to Home Page.
- Added the Homepage pattern.
- Added the Course List pattern.
- Added the Newsletter pattern.
- Added the Testimonial pattern.

When I viewed the page on the frontend, I was wondering why blocks were duplicated, then realized that the Homepage pattern was already including all those other patterns, and so I didn't need to add them separately. But it was confusing because the "Homepage" pattern preview image only shows the header.

Should we consider adding the pattern automatically when I select the Home Page template instead?

I think it would be wise to pull @burtrw and @Luchadores in at this point to get their early feedback, as this will also affect the Home Page Hero PR. Ronnie and Andrei, it would be appreciated if you could test this PR a bit to see if this is the flow you expect for creating a home page. These are the patterns this PR has added:

Screen Shot 2022-10-27 at 7 04 06 AM

**Comment by donnapep** [comment]
Let's not worry much about styling the footer as part of this PR, as there's a separate card for that.

**Comment by donnapep** [comment]
It looks like the Course List pattern could use a bit more testing. When none of my courses have images, the course list has a lot of extra whitespace, and there also appears to be extra whitespace above the title. Please let me know if you think this is an issue with the Course List block itself (in which case I'll open a separate Issue), or if it's related to this PR.

The Continue button is also not styled (and I'm guessing View Results won't be either). Please see[ my PR here](https://github.com/Automattic/sensei-theme/pull/52) for addressing button styles. I hope this PR can use the styles implemented there without needing to add its own:

Screen Shot 2022-10-27 at 7 24 53 AM


The styles for this PR appear to be affecting the Course List block on other pages. We'll need to isolate any styling changes to just the home page.

## Before
Screen Shot 2022-10-27 at 7 12 16 AM

## After
Screen Shot 2022-10-27 at 7 12 57 AM

File: patterns/testimonial-homepage.php [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"textColor":"tertiary"} -->
+<h3 class="has-tertiary-color has-text-color" style="text-transform:uppercase">What students say</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:columns {"style":{"spacing":{"blockGap":"20px"}},"textColor":"tertiary"} -->
+<div class="wp-block-columns has-tertiary-color has-text-color"><!-- wp:column {"verticalAlignment":"bottom","width":"41%"} -->
+<div class="wp-block-column is-vertically-aligned-bottom" style="flex-basis:41%"><!-- wp:image {"id":2150,"sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"8px","width":"1px"}}} -->
+<figure class="wp-block-image size-full has-custom-border"><img src="http://sentest.test/wp-content/themes/sensei-theme/assets/images/course-theme-testimonial.png" alt="" class="wp-image-2150" style="border-width:1px;border-radius:8px"/></figure>
```
Comment (gabrielcaires) [inline patterns/testimonial-homepage.php]: Replace `http://sentest.test` with `/`

File: patterns/homepage.php [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"20px","right":"20px","bottom":"100px","left":"20px"}}},"layout":{"type":"constrained","contentSize":"660px"}} -->
+<div class="wp-block-group alignfull" style="padding-top:20px;padding-right:20px;padding-bottom:100px;padding-left:20px"><!-- wp:spacer -->
+<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:paragraph {"align":"center","style":{"typography":{"textTransform":"uppercase","lineHeight":"1"}},"textColor":"primary","fontSize":"3-2x-large"} -->
+<p class="has-text-align-center has-primary-color has-text-color has-3-2-x-large-font-size" style="line-height:1;text-transform:uppercase">grow your writing Skills with Course</p>
```
Comment (gabrielcaires) [inline patterns/homepage.php]: Here should be the page title (I think`core/post-title`.)

File: patterns/footer.php [hunk 16]
```diff
@@ -7,6 +7,60 @@
...
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Spotify</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:heading {"textAlign":"center","fontSize":"5x-large","fontFamily":"sorts-mill-goudy"} -->
+<h2 class="has-text-align-center has-sorts-mill-goudy-font-family has-5-x-large-font-size">COURSE</h2>
```
Comment (gabrielcaires) [inline patterns/footer.php:60]: Here I think is the site title block

**Review by gabrielcaires — CHANGES_REQUESTED** [review]
In addition to the Donna's comments,  I found other small points.

**Comment by Luchadores** [comment]
Testing it right now on this Jurassic Ninja site: https://motionless-flea.jurassic.ninja/wp-admin/

- When I have no courses, in the course list block, the "No results found" is not visible enough as it's Green on Green almost. 

image

- Sidebar opened + switching the Course List block to the Grid View (which I would prefer to have selected by default, I got the list view by default), looks quite broken. (This is close to 1540px wide as a resolution, so there should be space to handle it better, I assume, maybe I'm wrong?).
image
Max resolution screenshot for grid view:
image

- Course List Block -> I also noticed the Start Course buttons are too wide compared to the rules we have for buttons. I would also limit the Course List body text to 3 lines of text. That should save us some vertical space.
image


- Clicking on the "Homepage" pattern, adds the entire homepage, which is great. But it's also not what I expected since I have the other single blocks listed as options there. I just expected to add the header and add the other blocks one by one. Maybe this is part of the test :) 
image

- When any of the sidebar menus are open, the background colors stick right to the edges of the content (Check the Course List block background to see what I mean in the screenshot below)
image

- The testimonial block renders with a broken image
- The testimonial block spacing for the author and profession is off. Check the block in the design (we should have 10px in-between the author name and profession).
image
image


**Mobile**

Switching to the mobile view in the editor renders a lot of spacing problems. I will add some screenshots below of what I'm seeing.

image
image
image
image
image
image



PS: This looks awesome to scroll through on a desktop! 👍  
PS2: Please check the [Figma file](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020040%3A18973) for spacing on both Desktop and Mobile.

**Comment by Imran92** [comment]
Thanks for taking a look @donnapep!

> Looks like the block pattern and template are named "Homepage" instead of "Home Page". I don't think we need to prefix each pattern with "Home Page", as it implies those patterns can only be used on a home page.

Changed it to Landing page and removed prefixes [here](https://github.com/Automattic/sensei-theme/pull/44/commits/0e9b310fb1d7d7db1274caed04eb1b97928fac4f)

> The testimonial image is not working for me, as the URL appears to be hardcoded.

Fixed as a part of [this commit](https://github.com/Automattic/sensei-theme/pull/44/commits/46592c3ecfd630ed62b365a18edd4411de136d34)

> When I viewed the page on the frontend, I was wondering why blocks were duplicated, then realized that the Homepage pattern was already including all those other patterns, and so I didn't need to add them separately. But it was confusing because the "Homepage" pattern preview image only shows the header.

Hopefully our Landing Page naming approach will help with that! :)

For course list related issues, as we've discussed, hopefully course list part will get fully fixed with the button styles and course list issue is completed! We'll probably need to think about some non ideal cases like when some courses have image and some don't, or all of them not having photos and come up with a solution to make it look good in all cases! ☺️ 

Thanks so much!

File: patterns/course-list-homepage.php (hunk 13 above)
Comment (Imran92) [reply patterns/course-list-homepage.php]: Updated it [here](https://github.com/Automattic/sensei-theme/pull/44/commits/d0018b883fc6a62fac73f6a30d24b3b370eac183)

File: patterns/blog-posts.php (hunk 12 above)
Comment (Imran92) [reply patterns/blog-posts.php]: Updated it [here](https://github.com/Automattic/sensei-theme/pull/44/commits/b73fbcff2137faa38f717e7d691953bc5072288d), Thanks!

File: patterns/testimonial-homepage.php (hunk 14 above)
Comment (Imran92) [reply patterns/testimonial-homepage.php]: Updated as a part of [this](https://github.com/Automattic/sensei-theme/pull/44/commits/46592c3ecfd630ed62b365a18edd4411de136d34), thanks!

File: patterns/homepage.php (hunk 15 above)
Comment (Imran92) [reply patterns/homepage.php]: I'm not sure we should use the page title here, as in farther discussion, we decided to use a template with no title. The user can edit it to put any text they want here

File: patterns/footer.php (hunk 16 above)
Comment (Imran92) [reply patterns/footer.php:60]: Thanks Gabriel! I think we should take a look at it separately under footer issue ☺️

**Comment by Imran92** [comment]
Thanks for taking a look @Luchadores ! I've updated the PR and hopefully the issues will be solved. The issues regarding the Course List will be handled in a separate PR. Please take a look when you can ☺️ And we've changed the name of the page template this time. Please along with the **Landing Page** pattern, use the **Page - No Title** template.

File: patterns/course-list.php [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase","lineHeight":"1"}},"fontSize":"medium","fontFamily":"league-gothic"} -->
+<h3 class="has-league-gothic-font-family has-medium-font-size" style="line-height:1;text-transform:uppercase">COURSE LIST</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|tertiary"}}}},"fontSize":"xx-small","fontFamily":"system"} -->
+<p class="has-link-color has-system-font-family has-xx-small-font-size"><a href="<?php echo class_exists( 'Sensei_Course' ) ? Sensei_Course::get_courses_page_url() : '#'; ?>">Explore all courses</a></p>
```
Comment (gabrielcaires) [inline patterns/course-list.php]: Translate string

File: patterns/course-list.php [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+<div class="wp-block-group has-tertiary-color has-primary-background-color has-text-color has-background" style="padding-top:0px;padding-right:0px;padding-bottom:40px;padding-left:0px"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}},"border":{"top":{"width":"0px","style":"none"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"color":"var:preset|color|tertiary","width":"1px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left","verticalAlignment":"top"}} -->
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase","lineHeight":"1"}},"fontSize":"medium","fontFamily":"league-gothic"} -->
+<h3 class="has-league-gothic-font-family has-medium-font-size" style="line-height:1;text-transform:uppercase">COURSE LIST</h3>
```
Comment (gabrielcaires) [inline patterns/course-list.php]: Translate string

**Comment by gabrielcaires** [comment]
I found some layout issues, my suggestion is to use this tool to compare the sizes, it changed my life:
https://chrome.google.com/webstore/detail/smart-page-ruler/nmibbjghlmdiafjolcphdggihcbcedmg

1) Should be 40px:
Screen Shot 2022-10-28 at 18 47 57

2) Should be 40px:
Screen Shot 2022-10-28 at 18 50 18

3) Author name aligned by middle
Screen Shot 2022-10-28 at 18 53 08


4) Category color should be white + margin-top = 20px
Screen Shot 2022-10-28 at 18 55 20

5) Should be 40


6) Should be 20
Screen Shot 2022-10-28 at 19 00 33


 7) Should be 40 (left)
Screen Shot 2022-10-28 at 19 03 06

I think that fixing it we can finally merge our cards

**Comment by Imran92** [comment]
Thanks for taking a look into it @gabrielcaires <3 I've addressed the spacing issues [here](https://github.com/Automattic/sensei-theme/pull/44/commits/90b07abb84bf6e2c3ee7dea8b1c37e82d8981069), and also as per our discussion in group, I've removed the course list related parts from this PR [here](https://github.com/Automattic/sensei-theme/pull/44/commits/36b0a95c776bc9ca64f00d41734b42a11be2149d), so could not address the comments regarding course list. Can you kindly take a look again? Thanks

File: patterns/newsletter-landing-page.php [hunk 19]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (gabrielcaires) [inline patterns/newsletter-landing-page.php:1]: is this file using the correct name? It doesn't seem like a full page.

**Review by gabrielcaires — APPROVED** [review]

**Pull request merged**